geode-core/src/main/java/org/apache/geode/management/internal/api/GeodeClusterManagementServiceConnectionConfig.java

```diff
@@ -44,62 +48,105 @@
 import org.apache.geode.internal.net.SocketCreatorFactory;
 import org.apache.geode.internal.security.SecurableCommunicationChannel;
 import org.apache.geode.logging.internal.log4j.api.LogService;
-import org.apache.geode.management.api.ClusterManagementService;
-import org.apache.geode.management.builder.ClusterManagementServiceBuilder;
-import org.apache.geode.management.internal.PlainClusterManagementServiceBuilder;
+import org.apache.geode.management.api.ClusterManagementServiceConnectionConfig;
+import org.apache.geode.management.client.ClusterManagementServiceBuilder;
 import org.apache.geode.management.internal.SSLUtil;
 import org.apache.geode.management.internal.configuration.messages.ClusterManagementServiceInfo;
 import org.apache.geode.management.internal.configuration.messages.ClusterManagementServiceInfoRequest;
 import org.apache.geode.management.internal.functions.GetMemberInformationFunction;
 import org.apache.geode.management.runtime.MemberInformation;
 import org.apache.geode.security.AuthInitialize;
 
+/**
+ * Concrete implementation of {@link ClusterManagementServiceConnectionConfig} which can be used
+ * to derive most (if not all) of the connection properties from an existing {@link Cache} or
+ * {@link ClientCache}.
+ *
+ * @see ClusterManagementServiceBuilder
+ */
+@Experimental
+public class GeodeClusterManagementServiceConnectionConfig
```

Comment:
this pretty much holds the same stuff as BasicClusterManagementServiceConnectionConfig. Is there anyway to make this delegate to that or inherit that?

Reply:
It now delegates.